File: aulas/aula 04/splitar.java
import java.util.ArrayList;

public class splitar {
    
    public static void main(String[] args) {
        ArrayList<glicemia> lista = new ArrayList<>();
        
        String linha = "120,11/03/2026,9:00";

        String vetorLinha[] = linha.split(",");

        glicemia obj = new glicemia(Integer.parseInt(vetorLinha[0]), vetorLinha[1], vetorLinha[2]);

        //System.out.println(obj.valor + " - " +obj.data + " - " + obj.hora);

        if (!lista.contains(obj)) {
            lista.add(obj);
        }

        for (glicemia item: lista) {
            System.out.println(item.valor + " - " + item.data + " - " + item.hora);
        }

    }

}

File: aulas/aula 04/splitar.py
from glicenia import glicenia
lista = []

linha = "120,11/02/2026,9:00"

vetor_linha = linha.split(",")

obj = glicenia(int(vetor_linha[0],vetor_linha[1],vetor_linha[2]))

if obj not in lista:
    lista.append(obj)
    